### ISSUE TITLE
sources\src\fmi is not included in the FMU

### ISSUE THREAD

**Issue by Nils12345678901234567** [NTNU-IHB/PythonFMU#214]
When the FMU is produced the source code is included in the directory named 'source' inside the .fmu file. There is also a CMakeList.txt file for building with CMake.  But to build the FMU the FMI header files are needed in the directory 'sources\src\fmi'. These are not included.

To be able to build the FMU from the source code the 'fmi' folder must be fetched from [PythonFMU/tree/master/pythonfmu/pythonfmu-export/src/fmi](https://github.com/NTNU-IHB/PythonFMU/tree/master/pythonfmu/pythonfmu-export/src/fmi).

**Comment by markaren** [comment]
Yes, this part of the software has never been tested or used as far as I know. I'm sure there are issues, like the one you report here. I guess it could be usefull to get a source code FMU working. A PR would be appriciated.

**Comment by markaren** [comment]
Closing as not supported
